apps/dashboard/app/(app)/apis/[apiId]/_components/create-key/components/key-created-success-dialog.tsx

```diff
@@ -74,7 +74,7 @@ export const KeyCreatedSuccessDialog = ({
       }}
     >
       <DialogContent
-        className="drop-shadow-2xl border-gray-4 overflow-hidden !rounded-2xl p-0 gap-0 min-w-[760px] max-h-[90vh] overflow-y-auto"
+        className="drop-shadow-2xl border-gray-4 overflow-hidden !rounded-2xl p-0 gap-0 min-w-[900px] max-h-[90vh] overflow-y-auto"
```

Comment:
<img width="1045" alt="image" src="https://github.com/user-attachments/assets/f0603324-5852-4e11-b85a-10475a5c54ad" />

I don't think we should change dialog widths